src/Compilers/VisualBasic/Portable/Binding/Binder_Invocation.vb

```diff
@@ -2424,7 +2424,7 @@ ProduceBoundNode:
                         ' Deal with Optional arguments
                         ' Need to handle optional arguments here, there could be conversion errors, etc.
 
-                        argument = GetArgumentForParameterDefaultValue(param, node, diagnostics, callerInfoOpt)
+                        argument = GetArgumentForParameterDefaultValue(param, node, diagnostics, callerInfoOpt, parameterToArgumentMap, arguments)
```

Comment:
>GetArgumentForParameterDefaultValue [](http://example.com/codeflow?start=35&length=35)

It looks like there is another call site for this method, but I didn't see it adjusted. Also, what happens for attribute applications.